neovim/api/common.py

```diff
@@ -21,6 +23,7 @@ def __init__(self, session, code_data):
         """
         self._session = session
         self.code_data = code_data
+        self.handle = unpackb(code_data[1])
```

Comment:
You could add `self.number = self.handle` here and the @property `number()` method above could be removed altogether.